ksp_plugin_test/vessel_test.cpp

```diff
@@ -39,18 +39,33 @@ class VesselTest : public testing::Test {
 
 using VesselDeathTest = VesselTest;
 
+TEST_F(VesselDeathTest, Uninitialized) {
+  EXPECT_DEATH(vessel_->history();, "is_synchronized");
```

Comment:
`EXPECT_DEATH`'s first parameter is called `statement` though, so this would rely on ill-documented behaviour. Let's go with the `();},` smiley.